### PR TITLE
SSBFFA: Add eelek's custom move

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
Extension of #196

File: mods/cssb/moves.js [hunk 1]
```diff
@@ -569,6 +569,50 @@ exports.BattleMovedex = {
...
+		},
+		secondary: {
+			chance: 20,
+			onHit: function (target, source) {
+				let result = this.random(3);
+				if (result === 0) {
+					target.trySetStatus('brn', source);
+				} else if (result === 1) {
+					target.trySetStatus('par', source);
+				//This is a bug. I dont know how to make sure only 2 events happen. if someone can fix it ill be really happe ~Eelek
```
Comment (HoeenCoder) [inline mods/cssb/moves.js]: `if (result === 0) { par } else { brn }`

Also im not getting anything but 0 or 1 when testing the rng for this bug. I rolled 3000 times.

Comment (ghost) [reply mods/cssb/moves.js]: At least I can remove that 3rd event

**Comment by ghost** [comment]
Fail

File: mods/cssb/moves.js [hunk 2]
```diff
@@ -598,16 +598,9 @@ exports.BattleMovedex = {
...
-					target.trySetStatus('brn', source);
-				} else if (result === 1) {
-					target.trySetStatus('par', source);
-				//This is a bug. I dont know how to make sure only 2 events happen. if someone can fix it ill be really happe ~Eelek
-				} else {
-					return false;
-				}
-			},
+				let result = this.random(2);
+				if (result === 0) { par } else { brn }
```
Comment (HoeenCoder) [inline mods/cssb/moves.js]: I didn't mean copy / paste exactly what i wrote >.>

Comment (ghost) [reply mods/cssb/moves.js]: I know that now

File: mods/cssb/moves.js [hunk 3]
```diff
@@ -598,7 +598,7 @@ exports.BattleMovedex = {
 		secondary: {
 			chance: 20,
 			onHit: function (target, source) {
-				let result = this.random(3);
```
Comment (HoeenCoder) [inline mods/cssb/moves.js]: use `this.random(2)`

**Comment by ghost** [comment]
Ready to merge @HoeenCoder

**Pull request merged**